app/Dockerfile

```diff
@@ -17,11 +17,21 @@ FROM python:3.10.10-bullseye
 # Create and run commands in /app inside the container
 WORKDIR /app
 
-# Copy the required files into the working directory
-COPY . /app/
+# Install the azure cli
+RUN apt-get update \
```

Comment:
can't be a requirement of the app because there is a dependency clash between latest cosmos sdk and azure-cli 😡